cli/src/main/scala/mdoc/internal/cli/CliEnrichments.scala

```diff
@@ -14,12 +14,16 @@ import coursierapi.Dependency
 import scala.meta.internal.io.FileIO
 import scala.meta.internal.inputs._
 import java.io.PrintStream
+import scala.util.Try
 
 trait CliEnrichments {
   implicit class XtensionInputMdoc(input: Input) {
     def filename: String =
       input match {
         case s: Input.Slice => s.input.filename
+        case f: Input.File => f.path.filename
+        case v: Input.VirtualFile =>
+          Try(Paths.get(v.path).getFileName().toString()).getOrElse(v.path)
```

Comment:
```suggestion
          Try(Paths.get(v.path).filename).getOrElse(v.path)
```

Reply:
Changed to  Try(AbsolutePath(Paths.get(v.path)).filename).getOrElse(v.path), since the extension method exists on AbsolutePath not Path. Alternative would be to add an extension for Path, but I don't think that is neccessary.